purchase_vendor_bill_breakdown/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module offers a solution for the following scenario:
+
+* We purchase a **Toolbox** product from a supplier
+* Supplier delivers us the Toolbox but issues an invoice with **Box**, **Wrench**, **Screwdriver** and **Hummer** products in line
+
+This module allows to replace a product in PO Line with a predefined set of components in the vendor bill line and also allows to calculate vendors product price by product components price.
```

Comment:
Please note: this module is not compatible with stock. To use these features with stock components, you need to install module purchase_stock_vendor_bill_breakdown .